extensions/sftp/processors/ListSFTP.cpp

```diff
@@ -359,21 +360,21 @@ bool ListSFTP::filterFile(const std::string& parent_path, const std::string& fil
 
   /* Age */
   time_t now = time(nullptr);
-  uint64_t file_age = (now - attrs.mtime) * 1000;
+  std::chrono::milliseconds file_age = std::chrono::seconds(now - attrs.mtime);
```

Comment:
```c++
  std::chrono::milliseconds file_age = std::chrono::file_clock::now() - std::chrono::seconds(attrs.mtime);
```

Reply:
this wouldnt work because the rhs would be a `time_point<file_clock>` and not a `duration`, and afaik the attrs.mtime is measured from unix epoch and since the epoch of the file_clock is not necessarily the same as the unix epoch, the `time_point<file_clock>::time_since_epoch` could return something completely different from what we want.

we could create a `time_point<file_clock>` from the attrs.mtime but for that we would need a platform specific helper function (because the file_clock interface is quite different on msvc, libc++ and on stdlibc++) it could be done with something similar to this (only in reverse):
https://github.com/rustammendel/nifi-minifi-cpp/pull/1/files#diff-796b46848475896d2982dcb681e0f547bac6aec3c98da80e8c03139562092604R89-R97

or we could stick to using system_clock (which has the unix epoch)
`auto file_age = std::chrono::system_clock::now() - std::chrono::system_clock::from_time_t(attrs.mtime);`

Reply:
Good point. I would go for `system_clock` in that case, for simplicity. I think that these timestamps are just system timestamps anyway.

Reply:
changed to `auto file_age = std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now() - std::chrono::system_clock::from_time_t(attrs.mtime));` (milliseconds because it will be printed out later) in https://github.com/apache/nifi-minifi-cpp/pull/1225/commits/78f8658fde802c73a5965dd734a544acbc37dca3